doc/source/whatsnew/v0.23.0.txt

```diff
@@ -365,7 +365,7 @@ Numeric
 Categorical
 ^^^^^^^^^^^
 
--
+- Bug in ``Categorical.equals`` between two unordered categories with the same categories, but in a different order (:issue:`16603`)
```

Comment:
use ``:func:`` here

Reply:
We don't list all the `Categorical` methods in the API docs, and `.equals` is one of the excluded ones.